sliders.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Uniter PHP Faders Demo</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="/assets/jquery-full.js"></script>
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body id="counting_elements">
+        <h1>
+            Uniter PHP Faders Demo
+        </h1>
+        <p>
+            The logic for this page is written entirely in PHP,
+            see <a href="//github.com/PharaohTools/uniter-jquery/blob/master/php/src/Sliders/app.php">Sliders</a>
+            for a look at the php code.
+        </p>
+        <!--<p>-->
```

Comment:
Most of these demos probably don't need tests (although it might be nice to refactor _some_ of them into classes to make them testable at some point), so we can probably delete this commented-out code (and the copies in the other `app.php` files)